mapdata/WarcraftLegacies/Units/-17920_-23552.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
@HerrDave Looks like this PR adds a bunch of new unit placements on the map. Is that intentional and if so, what are they?

Reply:
i think they are the testing units, i removed them